frontend/src/hooks/UserActions/Login/useLogin.ts

```diff
@@ -0,0 +1,32 @@
+import { Login } from './Login';
+import { useContext } from 'react';
+import { UserContext } from '../../../contexts/User/UserContext';
+
+export const useLogin = (): Login => {
```

Comment:
I can not fully understand what 'useLogin' name of the method stands for. You can try to think of a better name for that method.